### ISSUE TITLE
Faile to Use GPU In DeepLab Sample When using own tflite model On Android

### ISSUE THREAD

**Issue by DeepvisionsKorea** [asus4/tf-lite-unity-sample#102]
**Environment (please complete the following information):**
 - OS Version/Mobile: [Android 9] / [Galaxy S8]

**Describe the bug**
Hi, I'm very glad to use your tf-lite-unity-sample for my project.
It make progress on it.
However I met some problem on that.
My tflite model work in CPU option, but failed to use GPU opiton in DeepLabSample.
I thought it may be related with quantizaion option.
Would you give me some advice for adb message or chaninging model to quantized model?
(Well, It looks like there are many way  to change model quantized)
Here's Three Model's NETRON view.
1. is original DeepLabSample tflite of your git repo(deeplabv3_257_mv_gpu2)
it work very well
<img src="https://user-images.githubusercontent.com/76459523/109256694-e7103480-7839-11eb-96d1-2503bedfc7d3.png" width="50%" height="50%">

2.is another group's tflite model that work in GPU option
<img src="https://user-images.githubusercontent.com/76459523/109256751-060ec680-783a-11eb-881d-699fe159042b.png" width="50%" height="50%">

3.is my tflite model that doesn't work GPU option but CPU option
<img src="https://user-images.githubusercontent.com/76459523/109256831-30608400-783a-11eb-9177-e8fe466b67db.png" width="50%" height="50%">

+ adb message when I use GPU option when using my tflife model
<details>
	<summary>detail</summary>

--------- beginning of crash
--------- beginning of main
02-10 11:20:09.251 26673 26770 I Unity   : Camera2: CameraDevice closed.
02-10 11:20:09.256 26673 26691 D Unity   : Vulkan PSO: data extracted [new size=32722, old size=26508, incomplete=0]
02-10 11:20:09.258 26673 26691 D Unity   : Vulkan PSO: Saved size[32722] path[/storage/emulated/0/Android/data/com.DeepVisions.Wtflite/cache/vulkan_pso_cache.bin] success=1 headerChanged=0 sizeChanged=1
02-10 11:20:09.272 26673 26691 D Unity   : Sensor :        Accelerometer ( 1) ; 0.002394 / 0.00s ; LSM6DSL Acceleration Sensor / STM 
02-10 11:20:09.281 26673 26673 I Unity   : windowFocusChanged: false
02-10 11:20:09.290 26673 26691 D Unity   : SetWindow 0 0x0
02-10 11:20:10.155 26673 26673 I Unity   : onDestroy
02-10 11:21:42.363 27429 27490 I tflite  : Initialized TensorFlow Lite runtime.
02-10 11:27:20.798 28458 28458 D Unity   : CommandLine:  
02-10 11:27:20.858 28458 28458 I Unity   : onResume
02-10 11:27:20.947 28458 28492 D Unity   : SetWindow 0 0x718ffaf010
02-10 11:27:20.949 28458 28492 D Unity   : SetWindow 0 0x718ffaf010
02-10 11:27:21.021 28458 28458 I Unity   : windowFocusChanged: true
02-10 11:27:21.099 28458 28492 D Unity   : Enabling Unity systrace
02-10 11:27:21.151 28458 28492 D Unity   : [VFS] Mount /data/app/com.asus4.tfliteunitysample-mBkQ0v0DluAOmweQ0kgosQ==/base.apk
02-10 11:27:21.159 28458 28492 D Unity   : Loading player data from /data/app/com.asus4.tfliteunitysample-mBkQ0v0DluAOmweQ0kgosQ==/base.apk/assets/bin/Data/data.unity3d
02-10 11:27:21.188 28458 28492 I Unity   : SystemInfo CPU = ARM64 FP ASIMD AES, Cores = 8, Memory = 3679mb
02-10 11:27:21.188 28458 28492 I Unity   : SystemInfo ARM big.LITTLE configuration: 4 big (mask: 0xf0), 4 little (mask: 0xf)
02-10 11:27:21.189 28458 28492 I Unity   : ApplicationInfo com.asus4.tfliteunitysample version 0.1 build 0567510c-da80-4827-bb63-67da0f2e5172
02-10 11:27:21.189 28458 28492 I Unity   : Built from '2019.4/staging' branch, Version '2019.4.17f1 (667c8606c536)', Build type 'Development', Scripting Backend 'il2cpp', CPU 'arm64-v8a', Stripping 'Enabled'
02-10 11:27:21.200 28458 28492 D Unity   : Script Patching: Patch files are not available, '/storage/emulated/0/Android/data/com.asus4.tfliteunitysample/cache/ScriptOnly/2019.4.17f1/il2cpp/patch.config' is missing.
02-10 11:27:21.242 28458 28492 D Unity   : Same Unity build, not extracting il2cpp resources.
02-10 11:27:21.245 28458 28492 D Unity   : PlayerConnection initialized from /data/app/com.asus4.tfliteunitysample-mBkQ0v0DluAOmweQ0kgosQ==/base.apk/assets/bin/Data (debug = 0)
02-10 11:27:21.247 28458 28492 D Unity   : PlayerConnection initialized network socket : 0.0.0.0 55426
02-10 11:27:21.247 28458 28492 D Unity   : PlayerConnection initialized unix socket : Unity-com.asus4.tfliteunitysample
02-10 11:27:21.251 28458 28492 D Unity   : Multi-casting "[IP] 192.168.219.109 [Port] 55426 [Flags] 2 [Guid] 1309978080 [EditorId] 3122056024 [Version] 1048832 [Id] AndroidPlayer(samsung_SM-G950N@192.168.219.109) [Debug] 0 [PackageName] AndroidPlayer [ProjectName] tf-lite-unity-sample" to [225.0.0.222:54997]...
02-10 11:27:21.251 28458 28492 D Unity   : Started listening to [0.0.0.0:55426]
02-10 11:27:21.324 28458 28492 D Unity   : [EGL] Attaching window :0x718ffaf010
02-10 11:27:21.324 28458 28492 D Unity   : InitializeScriptEngine OK (0x7175842fc0)
02-10 11:27:21.324 28458 28492 D Unity   : PlayerConnection already initialized - listening to [0.0.0.0:55426]
02-10 11:27:21.382 28458 28492 D Unity   : Loading player data from assets/bin/Data/data.unity3d
02-10 11:27:21.388 28458 28492 D Unity   : Unable to lookup library path for 'libtensorflowlite_gpu_delegate', native render plugin support disabled.
02-10 11:27:21.389 28458 28492 E Unity   : Unable to find libtensorflowlite_gpu_delegate
02-10 11:27:21.404 28458 28492 D Unity   : Unable to lookup library path for 'libtensorflowlite_c', native render plugin support disabled.
02-10 11:27:21.405 28458 28492 E Unity   : Unable to find libtensorflowlite_c
02-10 11:27:21.411 28458 28492 D Unity   : Unable to lookup library path for 'libnnapi_delegate', native render plugin support disabled.
02-10 11:27:21.412 28458 28492 E Unity   : Unable to find libnnapi_delegate
02-10 11:27:21.427 28458 28492 D Unity   : Unable to load library '/data/app/com.asus4.tfliteunitysample-mBkQ0v0DluAOmweQ0kgosQ==/lib/arm64/libnnapi_delegate.so', native render plugin support disabled: java.lang.UnsatisfiedLinkError: dlopen failed: library "libc++_shared.so" not found
02-10 11:27:21.428 28458 28492 E Unity   : Unable to find nnapi_delegate
02-10 11:27:21.430 28458 28492 D Unity   : PlayerInitEngineNoGraphics OK
02-10 11:27:21.430 28458 28492 D Unity   : AndroidGraphics::Startup window =  0x718ffaf010
02-10 11:27:21.430 28458 28492 D Unity   : [EGL] Attaching window :0x718ffaf010
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: count=9
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_surface, enabled=1
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_android_surface, enabled=1
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_EXT_swapchain_colorspace, enabled=1
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_get_surface_capabilities2, enabled=0
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_EXT_debug_report, enabled=1
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_fence_capabilities, enabled=0
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_memory_capabilities, enabled=0
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_semaphore_capabilities, enabled=0
02-10 11:27:21.450 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_get_physical_device_properties2, enabled=0
02-10 11:27:21.472 28458 28492 D Unity   : [Vulkan init] SelectPhysicalDevice requestedDeviceIndex=-1 xrDevice=0x0
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] Physical Device 0x7173bfc0a8 [0]: "Mali-G71" deviceType=1 vendorID=13b5 deviceID=60a00001
02-10 11:27:21.473 28458 28492 D Unity   : Vulkan detection: 2
02-10 11:27:21.473 28458 28492 D Unity   : GfxDevice: creating device client; threaded=1
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: count=9
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_surface, enabled=1
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_android_surface, enabled=1
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_EXT_swapchain_colorspace, enabled=1
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_get_surface_capabilities2, enabled=0
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_EXT_debug_report, enabled=1
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_fence_capabilities, enabled=0
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_memory_capabilities, enabled=0
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_semaphore_capabilities, enabled=0
02-10 11:27:21.473 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_get_physical_device_properties2, enabled=0
02-10 11:27:21.511 28458 28492 D Unity   : [Vulkan init] SelectPhysicalDevice requestedDeviceIndex=-1 xrDevice=0x0
02-10 11:27:21.511 28458 28492 D Unity   : [Vulkan init] Physical Device 0x7173bfc0a8 [0]: "Mali-G71" deviceType=1 vendorID=13b5 deviceID=60a00001
02-10 11:27:21.511 28458 28492 D Unity   : [Vulkan init] Selected physical device 0x7173bfc0a8
02-10 11:27:21.511 28458 28492 D Unity   : [Vulkan init] Graphics queue count=1
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: count=20
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_incremental_present, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_16bit_storage, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_bind_memory2, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_dedicated_allocation, enabled=1
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_descriptor_update_template, enabled=1
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_fence, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_fence_fd, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_memory, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_semaphore, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_external_semaphore_fd, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_get_memory_requirements2, enabled=1
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_image_format_list, enabled=1
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_maintenance1, enabled=1
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_maintenance2, enabled=1
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_sampler_ycbcr_conversion, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_shader_draw_parameters, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_storage_buffer_storage_class, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_variable_pointers, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_EXT_queue_family_foreign, enabled=0
02-10 11:27:21.518 28458 28492 D Unity   : [Vulkan init] extensions: name=VK_KHR_swapchain, enabled=1
02-10 11:27:21.532 28458 28492 D Unity   : Vulkan API version 1.1.0 (1.0.97 supported by driver)
02-10 11:27:21.532 28458 28492 D Unity   : Vulkan vendor=[ARM] id=[13b5]
02-10 11:27:21.532 28458 28492 D Unity   : Vulkan renderer=[Mali-G71] id=[60a00001]
02-10 11:27:21.532 28458 28492 D Unity   : Vulkan device type 1
02-10 11:27:21.532 28458 28492 D Unity   : Vulkan driverversion=[0x04000000] uint=[67108864]
02-10 11:27:21.532 28458 28492 D Unity   : [vulkan] ASTC HDR support: 1
02-10 11:27:21.544 28458 28492 D Unity   : Vulkan PSO: Cache data successfully loaded [size=26380,path=/storage/emulated/0/Android/data/com.asus4.tfliteunitysample/cache/vulkan_pso_cache.bin]
02-10 11:27:21.545 28458 28492 D Unity   : [EGL] Attaching window :0x0
02-10 11:27:21.545 28458 28492 D Unity   : AndroidDisplayManagerVulkan::AttachWindow(0x718ffaf010, 0)
02-10 11:27:21.550 28458 28492 D Unity   : InitializeOrResetSwapChain 1080x2220 hdr=0 samples=0
02-10 11:27:21.606 28458 28492 D Unity   : AndroidDisplayManagerVulkan::AttachWindow(0x718ffaf010, 0)
02-10 11:27:21.606 28458 28492 D Unity   : Initialize engine version: 2019.4.17f1 (667c8606c536)
02-10 11:27:21.933 28458 28492 D Unity   : PlayerInitEngineGraphics OK
02-10 11:27:21.940 28458 28492 D Unity   : Found 34 native sensors
02-10 11:27:21.948 28458 28492 D Unity   : Sensor :        Accelerometer ( 1) ; 0.002394 / 0.00s ; LSM6DSL Acceleration Sensor / STM 
02-10 11:27:21.951 28458 28492 D Unity   : Sensor :        Accelerometer ( 1) ; 0.002394 / 0.00s ; LSM6DSL Acceleration Sensor / STM 
02-10 11:27:21.969 28458 28492 D Unity   : SetWindow 0 0x718ffaf010
02-10 11:27:21.969 28458 28492 D Unity   : AndroidDisplayManagerVulkan::AttachWindow(0x718ffaf010, 0)
02-10 11:27:21.979 28458 28492 D Unity   : InitializeOrResetSwapChain 1080x2220 hdr=0 samples=0
02-10 11:27:24.107 28458 28492 D Unity   : UnloadTime: 0.452846 ms
02-10 11:27:24.154 28458 28492 D Unity   : Sensor :        Accelerometer ( 1) ; 0.002394 / 0.00s ; LSM6DSL Acceleration Sensor / STM 
02-10 11:27:24.170 28458 28492 D Unity   : Choreographer available: Enabling VSYNC timing
02-10 11:27:24.207 28458 28492 I Unity   : jar:file:///data/app/com.asus4.tfliteunitysample-mBkQ0v0DluAOmweQ0kgosQ==/base.apk!/assets/nail1_float32.tflite
<pre><code>02-10 11:27:24.207 28458 28492 I Unity   : DeepLabSample:Start()
02-10 11:27:24.207 28458 28492 I Unity   :  
02-10 11:27:24.207 28458 28492 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
02-10 11:27:24.207 28458 28492 I Unity   : 
02-10 11:27:24.213 28458 28492 I tflite  : Created TensorFlow Lite delegate for GPU.
02-10 11:27:24.358 28458 28492 I tflite  : Initialized TensorFlow Lite runtime.
02-10 11:27:24.361 28458 28492 I tflite  : Created TensorFlow Lite XNNPACK delegate for CPU.
02-10 11:27:24.362 28458 28492 D Unity   : Unable to lookup library path for 'libc', native render plugin support disabled.
02-10 11:27:24.363 28458 28492 E Unity   : Unable to find libc
02-10 11:27:24.367 28458 28492 W Unity   : Interperter Warning: Attempting to use a delegate that only supports static-sized tensors with a graph that has dynamic-sized tensors. 
02-10 11:27:24.367 28458 28492 W Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
02-10 11:27:24.367 28458 28492 W Unity   : 
02-10 11:27:24.373 28458 28492 E Unity   : Exception: Failed to create TensorFlowLite Interpreter</code></pre>
02-10 11:27:24.373 28458 28492 E Unity   :   at TensorFlowLite.Interpreter..ctor (System.Byte[] modelData, TensorFlowLite.InterpreterOptions options) [0x00000] in <00000000000000000000000000000000>:0 
02-10 11:27:24.373 28458 28492 E Unity   :   at TensorFlowLite.BaseImagePredictor`1[T]..ctor (System.String modelPath, DeepLabSample sampler, System.Boolean useGPU) [0x00000] in <00000000000000000000000000000000>:0 
02-10 11:27:24.373 28458 28492 E Unity   :   at TensorFlowLite.DeepLab..ctor (System.String modelPath, UnityEngine.ComputeShader compute, DeepLabSample samplerr) [0x00000] in <00000000000000000000000000000000>:0 
02-10 11:27:24.373 28458 28492 E Unity   :   at DeepLabSample.Start () [0x00000] in <00000000000000000000000000000000>:0 
02-10 11:27:24.373 28458 28492 E Unity   :  
02-10 11:27:24.373 28458 28492 E Unity   : (Filename: currently not available on il2cpp Line: -1)
02-10 11:27:24.373 28458 28492 E Unity   : 
02-10 11:27:24.376 28458 28492 E Unity   : NullReferenceException: Object reference not set to an instance of an object.
02-10 11:27:24.376 28458 28492 E Unity   :   at DeepLabSample.Update () [0x00000] in <00000000000000000000000000000000>:0 
</details>

+ I try to change quantization option, but seems weird,,,(input int, output int, so doesn't work even in CPU option)
<img src="https://user-images.githubusercontent.com/76459523/109257022-a06f0a00-783a-11eb-9031-896450f79241.png" width="75%" height="75%">

Thank you, For Advice.

**Comment by asus4** [comment]
Hi @DeepvisionsKorea, I'm not sure, but there are two suspicious logs.

```
E Unity   : Unable to find libc
```

I've seen this error too, does this error occur when in CPU mode? Using the latest Unity 2019LTS might fix this, [see unity issue tracker](https://issuetracker.unity3d.com/issues/android-mono-il2cpp-unable-to-find-libc-error-thrown-when-executing-certain-sslstream-constructor?_ga=2.28223548.1404019401.1614248440-1255573165.1486024520&page=2#comments).

---

```
02-10 11:27:24.367 28458 28492 W Unity   : Interperter Warning: Attempting to use a delegate that only supports static-sized tensors with a graph that has dynamic-sized tensors. 
```

Your model's screenshot looks good. But please make sure that your model doesn't use the dynamic-sized tensor.

**Comment by DeepvisionsKorea** [comment]
Thank you for Advice. @asus4 
CPU also have same Error.
```
E Unity   : Unable to find libc
```
But create interpreter ignoring error.
And work in CPU mode.
Here's the adb logcat on CPU
It seems little different.

<details>
    <summary>adb logcat</summary>

--------- beginning of crash
--------- beginning of main
--------- beginning of system
02-27 14:30:21.816 15354 15378 I tflite  : Initialized TensorFlow Lite runtime.
02-27 14:48:57.617 16317 16335 I Unity   : SystemInfo CPU = ARM64 FP ASIMD AES, Cores = 8, Memory = 3679mb
02-27 14:48:57.617 16317 16335 I Unity   : SystemInfo ARM big.LITTLE configuration: 4 big (mask: 0xf0), 4 little (mask: 0xf)
02-27 14:48:57.621 16317 16335 I Unity   : ApplicationInfo com.DeepVisions.nail1_float32_CPU version 0.1 build 982909c9-229d-4cd4-aab2-0eea0c05e2b9
02-27 14:48:57.621 16317 16335 I Unity   : Built from '2019.4/staging' branch, Version '2019.4.17f1 (667c8606c536)', Build type 'Release', Scripting Backend 'il2cpp', CPU 'arm64-v8a', Stripping 'Enabled'
02-27 14:48:57.718 16317 16335 E Unity   : Unable to find libtensorflowlite_gpu_delegate
02-27 14:48:57.735 16317 16335 E Unity   : Unable to find libtensorflowlite_c
02-27 14:48:57.739 16317 16335 E Unity   : Unable to find libnnapi_delegate
02-27 14:48:57.745 16317 16335 E Unity   : Unable to find nnapi_delegate
02-27 14:49:00.218 16317 16335 I Unity   : jar:file:///data/app/com.DeepVisions.nail1_float32_CPU-cTWlzfOEmkA6oqijlR94oQ==/base.apk!/assets/nail1_float32.tflite
02-27 14:49:00.218 16317 16335 I Unity   : DeepLabSample:Start()
02-27 14:49:00.218 16317 16335 I Unity   :
02-27 14:49:00.218 16317 16335 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
02-27 14:49:00.218 16317 16335 I Unity   :
02-27 14:49:00.368 16317 16335 I tflite  : Initialized TensorFlow Lite runtime.
02-27 14:49:00.371 16317 16335 I tflite  : Created TensorFlow Lite XNNPACK delegate for CPU.
02-27 14:49:00.381 16317 16335 I Unity   : intput 0: name: Input, type: Float32, dimensions: [1,512,288,3], quantizationParams: {scale: 0 zeroPoint: 0}
02-27 14:49:00.381 16317 16335 I Unity   : output 0: name: Output, type: Int32, dimensions: [1,512,288,1], quantizationParams: {scale: 0 zeroPoint: 0}
02-27 14:49:00.381 16317 16335 I Unity   :
02-27 14:49:00.381 16317 16335 I Unity   : TensorFlowLite.InterpreterExtension:LogIOInfo(Interpreter)
02-27 14:49:00.381 16317 16335 I Unity   : TensorFlowLite.BaseImagePredictor`1:.ctor(String, Boolean)
02-27 14:49:00.381 16317 16335 I Unity   : TensorFlowLite.DeepLab:.ctor(String)
02-27 14:49:00.381 16317 16335 I Unity   : DeepLabSample:Start()
02-27 14:49:00.381 16317 16335 I Unity   :
02-27 14:49:00.381 16317 16335 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
02-27 14:49:00.381 16317 16335 I Unity   :
02-27 14:49:00.383 16317 16335 I Unity   : TensorflowLiteVersion 2.4.0
02-27 14:49:00.383 16317 16335 I Unity   : TensorFlowLite.BaseImagePredictor`1:.ctor(String, Boolean)
02-27 14:49:00.383 16317 16335 I Unity   : TensorFlowLite.DeepLab:.ctor(String)
<pre><code>
02-27 14:49:00.383 16317 16335 I Unity   : DeepLabSample:Start()
02-27 14:49:00.383 16317 16335 I Unity   :
02-27 14:49:00.383 16317 16335 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
02-27 14:49:00.383 16317 16335 I Unity   :
02-27 14:49:00.387 16317 16335 E Unity   : Unable to find libc
02-27 14:49:00.391 16317 16335 W Unity   : Interperter Warning: Attempting to use a delegate that only supports static-sized tensors with a graph that has dynamic-sized tensors.
02-27 14:49:00.391 16317 16335 W Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
02-27 14:49:00.391 16317 16335 W Unity   :
02-27 14:49:00.391 16317 16335 W Unity   : Interperter Warning: Ignoring failed application of the default TensorFlow Lite delegate indexed at 485301219744.
02-27 14:49:00.391 16317 16335 W Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)_
</code></pre>

</details>

And I used Unity 2019.4.17f, so I change v2019.4.20f and run example. 
Well..., I see logcat multiple time.
And I realized example model never make any error 
like this.
```
02-27 15:35:18.203 27388 27406 I Unity   : DeepLabSample:Start()
02-27 15:35:18.203 27388 27406 I Unity   :
02-27 15:35:18.203 27388 27406 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
```
But other models(my or other group's) which make some error message
Make own related error message and plus ```Unable to find libc```
```
02-27 15:43:59.895 30244 30260 I Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
02-27 15:43:59.895 30244 30260 I Unity   :
02-27 15:43:59.899 30244 30260 E Unity   : Unable to find libc
02-27 15:43:59.902 30244 30260 W Unity   : Interperter Warning: Attempting to use a delegate that only supports static-sized tensors with a graph that has dynamic-sized tensors.
02-27 15:43:59.902 30244 30260 W Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
02-27 15:43:59.902 30244 30260 W Unity   :
02-27 15:43:59.903 30244 30260 W Unity   : Interperter Warning: Ignoring failed application of the default TensorFlow Lite delegate indexed at 485298328032.
02-27 15:43:59.903 30244 30260 W Unity   : (Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
```
I'm sorry for this.
I agree your opinion. I should make sure not to use any-dynamic size tensor until the error message is gone and see everything(GPU) is fine...
